types/byte-size/byte-size-tests.ts

```diff
@@ -0,0 +1,22 @@
+import byteSize from 'byte-size';
+
+const toStringFn = () => {
+    return 'test';
+};
+
+byteSize(256, {
+    locale: 'en-gb',
+    precision: 1,
+    toStringFn,
+    units: 'iec_octet',
+});
+
+byteSize(12, {
```

Comment:
The tests are inadequate and don't cover all methods, e.g. `byteSize.defaultOptions`..